spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -6825,6 +6825,17 @@ TextMessage which throw JMSException.
 [appendix]
 == Change history
 
+=== Version 3.1
+* https://github.com/jakartaee/messaging/pull/311[issue_311] Bump `maven-bundle-plugin` version from 4.2.1 to 5.1.4
+* https://github.com/jakartaee/messaging/pull/306[issue_306] Remove useless parentheses
+* https://github.com/jakartaee/messaging/pull/298[issue_298] Repeatable annotation for the connection factory
+* https://github.com/jakartaee/messaging/pull/296[issue_296] Update `JMSDestinationDefinition`
+* https://github.com/jakartaee/messaging/pull/292[issue_292] Introduce new Ctors to set linked exception on creation
+* https://github.com/jakartaee/messaging/pull/291[issue_291] Version bump from 3.0 to 3.1
+* https://github.com/jakartaee/messaging/pull/289[issue_289] Typo fixed
+* https://github.com/jakartaee/messaging/pull/287[issue_287] Comment fixed
+* https://github.com/jakartaee/messaging/pull/284[issue_284] Minor updates to spec copyright and version referenced in javadoc
+
```

Comment:
What about
- #308 or #309

Reply:
That one wasn't marked with the 3.1 milestone, I've fixed that.
Also this one: https://github.com/jakartaee/messaging/pull/313